CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,61 @@
+# Changelog since v2.1.0
+
+## New Features
+
+### Snapshot APIs
+
+- The validation for volume snapshot objects (VolumeSnapshot and VolumeSnapshotContent) is getting more strict. Due to backwards compatibility this change will occur over multiple releases. The following key changes are highlighted.
+
+  1. As part of the first phase of the multi-phased release process, a validating
+webhook server has been added. This server will perform additional validation (strict) which was not done during the beta release of volume snapshots. It will prevent the cluster from gaining (via create or update) invalid objects.
+  2. The controller will label objects which fail the additional validation.
+
+  The combination of 1 and 2 will allow cluster admins to stop the increase of invalid objects, and provide a way to easily list all objects which currently fail the strict validation. It is the cluster admin's responsibility to install the webhook and to ensure all the invalid objects in the cluster have been deleted or fixed. See the KEP at https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/177-volume-snapshot/tighten-validation-webhook-crd.md ([#353](https://github.com/kubernetes-csi/external-snapshotter/pull/353), [@AndiLi99](https://github.com/AndiLi99))
```

Comment:
also clarify it's the k8s distribution/cluster admin responsibility.

Reply:
Done